.github/workflows/main.yml

```diff
@@ -6,18 +6,25 @@ jobs:
   check-pr:
     name: validate commits
     runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: ${{ github.event.pull_request.head.sha }}
+        fetch-depth: 0
+    - run: git fetch origin master
+    - uses: flux-framework/pr-validator@master
 
+
+  run-checks:
+    runs-on: ubuntu-latest
     strategy:
       matrix:
         python-version: [3.6, 3.7, 3.8]
```

Comment:
Do we still need to test against all three versions of python?  I'm assuming one would be sufficient now that this repo is no longer purely python. Thoughts?

Reply:
Thanks @SteVwonder; I agree. I just added a commit that drops the checks for `3.6` and `3.8`.